### PR TITLE
Email

### PR DESCRIPTION
**Pull request by elimelt**
Author: Elijah

## What changes were made?

Added a serializer and ask for email and name

### CODE REVIEW

File: src/components/debug/PairingDashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        toast({
+          title: 'Success',
+          description: 'Pair interview pool status updated',
+          status: 'success',
+          duration: 3000,
+          isClosable: true,
+        });
+      })
+      .catch((error) => {
+        console.error(error);
```
Comment (ShawnCollinge) [inline src/components/debug/PairingDashboard.tsx:38]: Let's put this behind devPrint or honestly toast might be nice.

File: src/App.tsx [hunk 2]
```diff
@@ -87,6 +88,16 @@ const App: React.FC = () => {
                 </ProtectedRoute>
               }
             />
+            <Route
+              path="/pairing"
+              element={
+                <ProtectedRoute>
+                  <DevRoute>
```
Comment (ShawnCollinge) [inline src/App.tsx:95]: Put this behind admin ? We can merge my pr then put admin instead of dev.

Comment (ShawnCollinge) [reply src/App.tsx:126]: You marked this as resolved but it' still devroute, do you want to leave it as devroute?

File: src/components/debug/PairingDashboard.tsx (hunk 1 above)
Comment (ShawnCollinge) [reply src/components/debug/PairingDashboard.tsx:38]: The console.error is still here :P I'm happy you added the toast though.

File: src/pages/AuthPage.tsx [hunk 3]
```diff
@@ -73,8 +74,13 @@ const AuthPage: React.FC = () => {
       return;
     }
 
+    if (!(discordUsername && username && email && password)) {
```
Comment (ShawnCollinge) [inline src/pages/AuthPage.tsx:77]: Do we want to add first name and last name?

Comment (ShawnCollinge) [reply src/pages/AuthPage.tsx:77]: Or we can just force first name and last name before signing up for interviews

Comment (elimelt) [reply src/pages/AuthPage.tsx:77]: yes first + last here is a good idea

File: src/App.tsx (hunk 2 above)
Comment (elimelt) [reply src/App.tsx:126]: no that was accident haha

File: src/components/debug/PairingDashboard.tsx (hunk 1 above)
Comment (ShawnCollinge) [reply src/components/debug/PairingDashboard.tsx:38]: 😡

Comment (ShawnCollinge) [inline src/components/debug/PairingDashboard.tsx:38]: 😡

Comment (elimelt) [reply src/components/debug/PairingDashboard.tsx]: oops

**Review by ShawnCollinge — APPROVED** [review]
LGTM

File: src/services/member.ts [hunk 4]
```diff
@@ -21,7 +21,21 @@ export function deserializeMember({
     discordUsername,
     resumeUrl,
     discordId,
-    groups: rest.groups.map((group) => group.name),
```
Comment (ShawnCollinge) [inline src/services/member.ts:24]: You broke admin with this 😢

**Pull request merged**